src/main/webapp/skin.xml

```diff
@@ -23,6 +23,8 @@
     xsi:schemaLocation="http://www.jasig.org/uportal/web/skin https://source.jasig.org/schemas/resource-server/skin-configuration/skin-configuration-v1.2.xsd">
 
 
+    <css>css/newsreader.css</css>
```

Comment:
I attempted to add the stylesheet to skin.xml so it would not be referenced on every JSP but could not get my application to render it. This should be addressed before merging.